module/VuFind/src/VuFind/Db/Entity/UserInterface.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * Entity model interface for users.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
+ */
+
+namespace VuFind\Db\Entity;
+
+use VuFind\Db\Interface\UserAccountInterface;
+
+/**
+ * Entity model interface for users.
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
+ */
+interface UserInterface extends UserAccountInterface
```

Comment:
This should not be necessary. I plan to add all of these methods to the existing UserAccountInterface in the near future -- I just need to wait for #3485 to be merged, and then I plan to begin a new PR to begin expanding the UserAccountInterface and using it in more places.

Reply:
I agree, I was expecting to change this one way or another based on your review.